lib/index.js

```diff
@@ -192,6 +202,37 @@ KnexMigrator.prototype.migrateTo = function migrateTo(options) {
     });
 };
 
+/**
+ * will delete the target database
+ *
+ * @TODO:
+ * - think about deleting only the tables
```

Comment:
> 1. deleting and recreating the whole database (and connection) between every test will result in slower tests for not much value

Yeah, let me test that again. I started with deleting all tables for mysql, but i saw it was not slower. So i decided to just drop the whole database. I will make a whole test run in Ghost with both solutions and then report here. 

> 2. having "reset" delete the database steps a little outside of what the migrator tool possibly ought to have control over. What I mean is, the init & upgrade migrations only ever affect the database in a way set out in the corresponding files. This reset command will delete EVERYTHING regardless of whether it was ever created by knex-migrator. It's an edgecase, but people do and will install Ghost and then add their own stuff manually to the database. I think deleting everything in this case would be unexpected?

Ok i see. I think we can offer the option to override the reset command. For example: you can add a reset file into your migrations path. And if this exists, we will run your own logic. So we can use that in Ghost later if we want. What do you think?I would like to keep the reset logic, might be helpful for other projects.